manifests/j/JackMordaunt/icnsify/2.2.6/JackMordaunt.icnsify.locale.en-US.yaml

```diff
@@ -0,0 +1,37 @@
+# Created with WinGet Releaser v2 using Komac v1.5.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: JackMordaunt.icnsify
+PackageVersion: 2.2.6
+PackageLocale: en-US
+Publisher: Jack Mordaunt
+PublisherUrl: https://liberapay.com/JackMordaunt

```

Comment:
```suggestion
# PublisherUrl: https://liberapay.com/JackMordaunt
```

Reply:
Why should this be commented?

Reply:
It's just one way to get past the 403 status code causing the URL Error